how-to-guides/restricting-signup.md

```diff
@@ -0,0 +1,48 @@
+---
+description: "Completely disable signing up of new users on Appsmith or restrict the domains of email address that can be used to sign-up."
+---
+
+# Restricting Sign-up
+
+## Disable sign-up
```

Comment:
@sharat87 can you add steps on where to add these environment variables and how to restart the docker instance

Reply:
Also can you link to this in the setup step, we already have a section on disabling signup there

Reply:
Pushed both changes.